packages/gatsby-source-shopify/src/queries.js

```diff
@@ -224,3 +224,21 @@ export const PRODUCT_TYPES_QUERY = `
     }
   }
 `
+
+export const PAGES_QUERY = `
+  query GetPages($first: Int!) {
```

Comment:
shouldn't we support the same parameters as the other queries?
https://help.shopify.com/en/api/custom-storefronts/storefront-api/reference/queryroot#pages

```suggestion
  query GetPages($first: Int!, $after: String) {
```